cors/status-async.htm

```diff
@@ -60,36 +61,37 @@ <h1>Status returned</h1>
     if (expect_code === undefined)
         expect_code = code
 
-    async_test("Status on " + method + " " + code + (nonsimple?' (nonsimple)':''), { timeout: 15000 })
-    .step(function() {
-        var client = new XMLHttpRequest()
-
-        client.open(method, CROSSDOMAIN + "resources/status.py?code="
-          + code + '&headers=x-nonsimple&text=OHAI', true)
-
-        if (nonsimple)
-            client.setRequestHeader('x-nonsimple', true)
-
-        client.onreadystatechange = this.step_func(function() {
-            if (client.readyState < client.HEADERS_RECEIVED)
-                return
-            assert_equals(client.response, "", "response data")
-            assert_equals(client.status, expect_code, "response status")
-            /* Response code 200 forces webserver to send OK(?) */
-            if(expect_code == 200)
-                assert_equals(client.statusText, "OK", "response statusText")
-            else
-                assert_equals(client.statusText, (expect_code == 0 ? "" : "OHAI"), "response statusText")
-            if (client.readyState == client.DONE)
-                this.done()
+    async_test(function() {
```

Comment:
This seems wrong? Did you rerun all these tests and verify the results are identical?

Reply:
Yes, I'm seeing the same results on localhost (http://web-platform.test:8000) as I see when I run the tests on http://w3c-test.org/. Which part specifically seems wrong?

#### Local tests using this branch
![screen shot 2018-07-17 at 2 07 34 pm](https://user-images.githubusercontent.com/54056/42837188-5dec42f2-89cb-11e8-96fe-5dcbc4d23e50.png)


#### w3c-test.org using master
![screen shot 2018-07-17 at 2 08 01 pm](https://user-images.githubusercontent.com/54056/42837179-5649d8ca-89cb-11e8-9459-2e6158831448.png)